File: API/WebApplication1/Models/Employee.cs
using System;
using System.Collections.Generic;

namespace WebApplication1.Models
{
    public class Employee
    {
        public int Id { get; set; }
        public string FirstName { get; set; }
        public string FatherName{ get; set; }
        public string LastName { get; set; }
        public string Fin { get; set; }
        public string InjuranceNo { get; set; }
        public DateTime BirthDate { get; set; }
        public DateTime StartDate { get; set; }
        public string CommandNo { get; set; }
        public string PTMoney  { get; set; }
        public string Meharetlilik { get; set; }
        public string Temsilcilik { get; set; }
        public string Mexfilik { get; set; }
        public string Zererlilik { get; set; }
        public string XariciDil  { get; set; } 
        public string Kesfiyyat { get; set; }
        public string ElmiDerece { get; set; }
        public string FexriAd { get; set; }
        public string ExtraMoney { get; set; }
        public string ExtraMoney2 { get; set; }        
        public int? RankId { get; set; }
        public virtual Rank Rank { get; set; }
        public int? PositionId { get; set; }
        public virtual Position Position { get; set; }
        public ICollection<EmployeeSalaryRecord> EmployeeSalaryRecords { get; set; }
    }
}

File: Frontend/src/app/constants/headers/employee.ts
import { Column } from "react-table";
import { EmployeeData } from "../../types/EmployeeData";
import moment from "moment";

export const employeeHeaders: Column<EmployeeData>[] = [
    {
      Header: "Soyadı",
      accessor: "lastName",
    },
    {
      Header: "Adı",
      accessor: "firstName",
    },
    {
      Header: "Atasının adı",
      accessor: "fatherName",
    },
    {
      Header: "Sosial Sığorta №",
      accessor: "injuranceNo",
    },
    {
      Header: "Doğum tarixi",
      accessor: (row) => moment(new Date(row.birthDate)).format("DD.MM.YYYY") || "",
    },
    {
      Header: "FIN",
      accessor: "fin",
    },
    {
      Header: "Təyin olunma",
      columns: [
        {
          Header: "vaxtı",
          accessor: (row) => moment(new Date(row.startDate)).format("DD.MM.YYYY") || "",
        },
        {
          Header: "əmri",
          accessor: "commandNo",
        },
      ],
    },
    {
      Header: "Qəbul olunma",
      columns: [
        {
          Header: "vaxtı",
          id: "enteranceDate",
          accessor: (row) => moment(new Date(row.enteranceDate)).format("DD.MM.YYYY") || "",
        },
        {
          Header: "əmri",
          accessor: "enteranceCommand",
        },
      ],
    },
    {
      Header: "Rütbənin dəyişdirilməsi",
      accessor: (row) => row.rank?.shortName || "",
    },
    {
      Header: "Vəzifənin dəyişdirilməsi",
      accessor: (row) => row.position?.name || "",

    },
    {
      Header: "Məharət dərəcəsi",
      columns: [
        {
          Header: "verilmə tarixi",
        },
        {
          Header: "Məharət dərəcəsi",
          // accessor: "meharetlilik",
        },
      ],
    },
    {
      Header: "Təmsilçilik",
      accessor: "temsilcilik",
    },
    {
      Header: "Məxfilik",
      accessor: "mexfilik",
    },
    {
      Header: "Zərərliyə görə",
      accessor: "zererlilik",
    },
    {
      Header: "Xarici dil",
      accessor: (row) => row.xariciDil?.name || "",
    },
    {
      Header: "Elmi dərəcə",
      accessor: "elmiDerece",
    },
    {
      Header: "Fəxri ad",
      accessor: (row) => row.fexriAd?.name || "",
    },
  ];